src/listen/websocket.rs

```diff
@@ -238,12 +257,60 @@ fn options_to_query_string(options: &Options) -> String {
     serde_urlencoded::to_string(serialized_options).unwrap_or_default()
 }
 
+#[derive(Debug)]
+pub struct TranscriptionStream {
+    write_arc: Arc<Mutex<SplitSink<WebSocketStream<MaybeTlsStream<TcpStream>>, Message>>>,
+}
+
+impl TranscriptionStream {
+    pub async fn finalize(
+        &self,
+        event_tx: Sender<Event>,
+    ) -> std::result::Result<(), DeepgramError> {
+        let finalize_message = Message::Text("{\"type\": \"Finalize\"}".to_string());
```

Comment:
Idiomatic rust suggestion: use raw strings to avoid escaping quote characters.  A rust raw string consists of an `r` followed by zero or more `#`s (usually one) followed by a double quote, then the string, then the closing quote, and finally the same number of `#`s as at the start of the string.  Allowing any number of `#`s means you can include strings that contain `"##` sequences of any length.  So for example:
```rust
let string = r###"This is a r##"raw string that contains a r#"raw "string""# and a r"raw string with a \n" so when it ends here"## the text is still part of one big string"###;

But more simply, and more normally:

```suggestion
        let finalize_message = Message::Text(r#"{"type": "Finalize"}"#.to_string());
```

Reply:
Resolved